homework4/task7.js

```diff
@@ -0,0 +1,15 @@
+function infiniteSum(arg) {
+    let total = 0;
+    function inner(arg) {
+        console.log(typeof arg)
+        if (typeof arg === "function") {
+            arg(total);
+        } else {
+            total += arg;
+            return inner;
+        }
+    }
+    
+    return inner(arg);
+}
+infiniteSum(5)(3)(4)(result => {console.log(result)})
```

Comment:
INCORRECT

You should return the function result with getter:
E.G. infiniteSum(6)(2)(4)(12).get()